src/GUI/UI/UIHouses.h

```diff
@@ -40,6 +38,24 @@ bool houseDialogPressEscape();
  */
 void playHouseSound(HOUSE_ID houseID, HouseSoundType type);
 
+enum class HouseInteractiveType {
+    HOUSE_INTERACTIVE_PROPRIETOR,
+    HOUSE_INTERACTIVE_NPC,
+    HOUSE_INTERACTIVE_TRANSITION
+};
+using enum HouseInteractiveType;
```

Comment:
Let's add doxydocs. E.g. what's the difference between proprietor and npc? Let's write this out here.

Reply:
Also, why don't we name this HouseNpcType / HouseNpcDesc? Transitions are essentially "fake" npcs after all.